float64[6] cartesian_stiffness
--------------------------


